apps/bluetooth/js/modules/bluetooth/bluetooth_adapter_manager.js

```diff
@@ -15,7 +15,7 @@ define(function(require) {
   var Observable = require('modules/mvvm/observable');
 
   var _debug = false;
-  var Debug = function() {};
+  var debug = function() {};
   if (_debug) {
     Debug = function btam_debug(msg) {
```

Comment:
debug instead of Debug here, and you'll see some error if following code does not replaced..